docs/guide/db-active-record.md

```diff
@@ -667,7 +667,7 @@ public function actionUpdate($id)
 
     try {
         if ($model->load(Yii::$app->request->post()) && $model->save()) {
-            return $this->redirect(['view', <?= $urlParams ?>]);
+            return $this->redirect(['view', $urlParams]);
```

Comment:
should be something like `['id' => $model->primaryKey]` instead of `$urlParams` because this variable is not defined.